### ISSUE TITLE
ADVECTION_IMPLICIT_MANUFACTURED_SOLUTION_STRONG Fails

### ISSUE THREAD

**Issue by cpethrick** [dougshidong/PHiLiP#204]
Since merging PR #111, these tests are expected to fail. The changes made to strong mean that we can no longer use AD, and therefore cannot solve DG Strong using the implicit ODE solver type.

**Comment by jbrillon** [comment]
Could be fixed if we change it to use implicit RK once that's implemented

**Comment by dougshidong** [comment]
Implicit RK should converge with an infinite time-step. Advection equation is linear, and thus unconditionally stable with implicit schemes.

**Comment by cpethrick** [comment]
I agree that it's unconditionally stable. However, the JFNK implicit solver currently doesn't have a preconditioner, so the linear solve might not converge. Wouldn't that limit the step size?

**Comment by dougshidong** [comment]
Interesting. I'm gonna guess that 1D just doesn't hit the tolerance required. I'm fine to keep this issue open. This would get fixed by being able to AD strong form if we ever do.
